### ISSUE TITLE
Linking to auto-updated releases

### ISSUE THREAD

**Issue by azeemba** [walkerdb/supreme_court_transcripts#2]
Thanks a lot for creating this and documenting the API.

I was able to fork this repo and use GitHub Actions to set up auto-updating of the cases and also of publishing weekly releases for quick downloads.

Would you be interested in linking to those updated releases? The link for that is https://github.com/azeemba/supreme_court_transcripts_auto_updated/releases

If you are interested, I would be happy to send a PR as well. The only caveat is that I have about 10 additional commits that renamed some existing files so they were consistent with the naming scheme the updating script follows (it was mostly making sure docket number in the file name matched what Oyez said it was).

**Comment by walkerdb** [comment]
Hello,

These changes are wonderful, I'd love to add them in! I've also sent an invite to add you as a collaborator -- please feel free to update / clean things up as you'd like.

**Comment by azeemba** [comment]
Thanks for the access!

I have merged in my changes here. The [Actions](https://github.com/walkerdb/supreme_court_transcripts/actions) tab shows the AutoUpdate and AutoPublish jobs which are currently scheduled to run on Saturdays. So we should have validation next week to confirm that everything moved over fine.

In the meanwhile, I have manually created a release so that the link in the README.md still leads to a valid release.

Will close the issue and archive my fork since this issue has been resolved.
